README.md

```diff
@@ -1,48 +1,105 @@
-# Snowflake Native SDK for Connectors
+# Native SDK for Connectors
 
-## About the Snowflake Native SDK for Connectors
+Data is fuel for data clouds. Transferring the data to Snowflake is a step where the whole data journey begins so that 
+is why, this step on the Snowflake customer's path is such an essential one.
```

Comment:
I would repharase a bit: 
```suggestion
Data is fuel for data clouds. The journey of data begins with transferring it into Snowflake, because of that this is essential step for customers.
```
Or:
"Transferring the data to Snowflake is where the data journey begins, this is why this step on the Snowflake customer's path is such an essential one."